### PR TITLE
using "_".getBytes() as a constant

### PR DESCRIPTION
**Pull request by benediamond-lab**
**What does this PR do?**

**Why are these changes required?**

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

**Comment by shydesky** [comment]
@benediamond-lab please fix the sonar problem. 
https://sonarcloud.io/project/issues?id=java-tron&open=AW_Gw0_tEUkFdNAuEQ7N&pullRequest=2901&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL

File: actuator/src/main/java/org/tron/core/actuator/ExchangeCreateActuator.java [hunk 1]
```diff
@@ -59,14 +61,14 @@ public boolean execute(Object object) throws ContractExeException {
 
       accountCapsule.setBalance(newBalance);
 
-      if (Arrays.equals(firstTokenID, "_".getBytes())) {
+      if (Arrays.equals(firstTokenID, TRX_SYMBOL_BYTES)) {
```
Comment (jiangyy0824) [inline actuator/src/main/java/org/tron/core/actuator/ExchangeCreateActuator.java]: Naming TRX_SYMBOL_BYTES is not appropriate. Please use more direct words. For example underline or undercode.

Comment (benediamond-lab) [reply actuator/src/main/java/org/tron/core/actuator/ExchangeCreateActuator.java]: ok

Comment (shydesky) [reply actuator/src/main/java/org/tron/core/actuator/ExchangeCreateActuator.java]: > Naming TRX_SYMBOL_BYTES is not appropriate. Please use more direct words. For example underline or undercode.

what is the meaning of "undercode"? Are you sure?  The character "_" stands for TRX when creating a transaction of ExchangeCreate. IMO, the TRX_SYMBOL_BYTES or TRX_SYMBOL is the direct word.

Comment (benediamond-lab) [reply actuator/src/main/java/org/tron/core/actuator/ExchangeCreateActuator.java]: agree!

**Pull request closed without merging**